src/shared/components/FluxMonacoEditor.scss

```diff
@@ -51,22 +51,26 @@
   .sync-bar {
     position: absolute;
     width: 48px;
-
-    .sync-icon {
-      color: #0098f0;
-    }
   }
 
   .sync-bar--on {
-    background: #0098f01a; // blue with 10% opacity
-    border-top: 1px solid #0098f0;
-    border-bottom: 1px solid #0098f0;
+    background: #00a3ff26; // blue with 15% opacity
```

Comment:
I left this as hex code since it requires a 15% opacity. `26` at the end is the hex value for 15% opacity